instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -84,17 +85,25 @@ class PostsPage extends Component {
     this.setState({data:comments})
   }
 
+  
   render() {
+    const PostDiv = styled.div`
+    @media(max-width:700px){
```

Comment:
This is as well not right. Try to leave your render function, cleanest as possible. Stuffing it with code that shouldn't be there will lead you to a bunch of unexpected bugs. 
This is indeed part of the problem with your input fields bug.